### PR TITLE
feat: add event for September 2024

### PR DESCRIPTION
**Pull request by astorije**
## PR Checklist

- [x] Fixes https://github.com/SquiggleTools/boston-ts-website/issues/341
- [x] That issue was marked as [`status: accepting prs`](https://github.com/JoshuaKGoldberg/boston-ts-website/issues?q=is%3Aopen+is%3Aissue+label%3A%22status%3A+accepting+prs%22)
- [x] Steps in [CONTRIBUTING.md](https://github.com/JoshuaKGoldberg/boston-ts-website/blob/main/.github/CONTRIBUTING.md) were taken

## Overview

Add event for September 2024

### CODE REVIEW

File: app/data/events.json [hunk 1]
```diff
@@ -1,4 +1,40 @@
 [
+	{
+		"date": "2024-09-29T18:00:00.000-04:00",
+		"link": "https://www.eventbrite.com/e/boston-ts-club-vi-rise-of-the-squiggles-tickets-1014606444187",
```
Comment (astorije) [inline app/data/events.json:4]: I am very upset this event is named after SW episode 9 and not episode 6 "Return of the Jedi".

File: app/data/events.json [hunk 2]
```diff
@@ -1,4 +1,40 @@
...
+				"author": {
+					"name": "Josh Goldberg",
+					"url": "https://joshuakgoldberg.com/"
+				},
+				"title": "What's New in TypeScript 5.5"
+			},
+			{
+				"author": {
+					"name": "Paula Paul",
+					"url": ""
```
Comment (astorije) [inline app/data/events.json:18]: @JoshuaKGoldberg do you happen to have URLs for these speakers?

File: cspell.json [hunk 3]
```diff
@@ -10,27 +10,27 @@
...
+	"overrides": [
+		{
+			"filename": "**/*.css.ts",
+			"languageId": "css"
+		}
+	],
+	"dictionaryDefinitions": [
+		{
+			"name": "speaker-names",
+			"path": "./cspell.speaker-names.txt"
```
Comment (astorije) [inline cspell.json:33]: Thoughts, is this overkill?

File: cspell.json [hunk 4]
```diff
@@ -10,27 +10,27 @@
...
-		"Blass",
 		"bluesky",
 		"bostonts",
-		"clsx",
-		"currentcolor",
-		"Dimitri",
-		"Farolino",
 		"Hadley",
-		"isbot",
+		// "isbot",
```
Comment (astorije) [inline cspell.json:17]: What is this for?

File: cspell.json [hunk 5]
```diff
@@ -10,27 +10,27 @@
...
-		"Ndukwe",
-		"Nnenna",
 		"packagejson",
 		"Sanker",
 		"Supabase"
+	],
+	"overrides": [
+		{
+			"filename": "**/*.css.ts",
+			"languageId": "css"
```
Comment (astorije) [inline cspell.json:27]: This allowed me to remove `currentcolor` from the word list. Big whoop.

File: app/data/events.json (hunk 1 above)
Comment (JoshuaKGoldberg) [reply app/data/events.json:4]: @astorije if you change it here I'll be happy to change it on Eventbrite 😄

File: app/data/events.json (hunk 2 above)
Comment (JoshuaKGoldberg) [reply app/data/events.json:18]: https://x.com/paulapaultweets?lang=en

https://x.com/corinamurg?lang=en

https://blackgirlbytes.dev/

File: cspell.json (hunk 4 above)
Comment (JoshuaKGoldberg) [reply cspell.json:29]: Oh right it was some internal thing used by Remix for the Philly JS Club site. I don't know if it's still needed.

File: cspell.speaker-names.txt
Comment (JoshuaKGoldberg) [inline cspell.speaker-names.txt:1]: I like this a lot! Good call.

**Review by JoshuaKGoldberg — APPROVED** [review]
✨🙌

File: app/data/events.json (hunk 1 above)
Comment (astorije) [reply app/data/events.json:4]: Return of the Squiggle?

File: app/data/events.json (hunk 2 above)
Comment (astorije) [reply app/data/events.json:18]: Amazing, thank you!

File: app/data/events.json (hunk 1 above)
Comment (JoshuaKGoldberg) [reply app/data/events.json:4]: Let's do it.

File: cspell.json (hunk 4 above)
Comment (astorije) [reply cspell.json]: Welp, I guess it was indeed necessary: https://github.com/SquiggleTools/boston-ts-website/actions/runs/10874012817/job/30170816313?pr=340

Comment (JoshuaKGoldberg) [reply cspell.json]: lol.

File: app/data/events.json (hunk 1 above)
Comment (astorije) [reply app/data/events.json]: Done. Note that I went with singular, not plural. [There seems to be controversy](https://www.google.com/search?q=%22return+of+the+jedi%22+singular+or+plural&oq=%22return+of+the+jedi%22+singular+or+plural&gs_lcrp=EgZjaHJvbWUyBggAEEUYOTIHCAEQIRigATIHCAIQIRigATIHCAMQIRigATIHCAQQIRigATIHCAUQIRirAtIBCDY0NTFqMGo3qAIAsAIA&sourceid=chrome&ie=UTF-8) however so I'm happy to change 😅

**Pull request merged**